extensions/book/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "book",
+    "displayName": "book",
```

Comment:
>book [](start = 20, length = 4)

Display names & descriptions should be localized - see [https://github.com/microsoft/azuredatastudio/blob/master/extensions/admin-tool-ext-win/package.json](https://github.com/microsoft/azuredatastudio/blob/master/extensions/admin-tool-ext-win/package.json) and [https://github.com/microsoft/azuredatastudio/blob/master/extensions/admin-tool-ext-win/package.nls.json](https://github.com/microsoft/azuredatastudio/blob/master/extensions/admin-tool-ext-win/package.nls.json) for how this is set up for package.json